MyHabits/HabitsViewController + Cell/ProgressHeaderView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ProgressCollectionViewCell.swift
+//  MyHabits
+//
+//  Created by Anton Kislov on 08.06.2023.
+//
+
+import UIKit
+
+class ProgressHeaderView: UICollectionReusableView {
+    
+    lazy var cellContentView: UIView = {
+        let view = UIView()
+        view.layer.cornerRadius = 8
+        view.backgroundColor = .white
+        view.contentMode = .scaleAspectFill
+        view.clipsToBounds = true
+        return view
+    }()
+    
+    lazy var motoLable: UILabel = {
+        let label = UILabel()
+        label.font = const.footnoteSmallFont
+        label.textColor = const.sistemGray
+        label.textAlignment = .left
+        return label
+    }()
+    
+    lazy var persentLabel: UILabel = {
+        let label = UILabel()
+        label.font = const.footnoteSmallFont
+        label.textColor = const.sistemGray
+        label.textAlignment = .right
+        return label
+    }()
+    
+    lazy var slider: UISlider = {
```

Comment:
check
https://developer.apple.com/documentation/uikit/uiprogressview

Usually you would use slider if user interacts with it